src/main/webapp/js/omniComment.js

```diff
@@ -86,16 +86,16 @@ $(document).ready(function() {
             $('#no-comment-panel').show();
             // if all is checked, show giver and status for better user experience
             if (!$('#panel_all').prop('checked')) {
-                $('#giver_all').parent().parent().hide();
-                $('#status_all').parent().parent().hide();
+                $('#giver_all').closest('.col-sm-4').hide();
```

Comment:
I would really prefer adding something like that than selecting a commonly used bootstrap class 👍